### PR TITLE
Corrected error response descriptions

### PR DESCRIPTION
**Pull request by MikeAdamsMews**
* Corrected error response descriptions in `responses.md` to include both `error` and `errors` properties
* Added fourth example to illustrate the use of `errors`
* Minor improvement to wording at the head of the page
* Ref. Asana ticket https://app.asana.com/0/1201871773388036/1207098479272602/f

### CODE REVIEW

File: guidelines/responses.md [hunk 1]
```diff
@@ -61,11 +61,32 @@ See the [Error codes](#error-codes) table below for further details about specif
...
+   ]
+}
+```
+
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `success` | `bool` | required | Determinines the result of the operation. |
 | `asyncConfirmation` | `bool` | optional | Determinines if there will be following asynchronous response. |
-| `errors` | array of [`Error`](#error) | optional | In case of `"success": false`, this property holds information about the errors that occurred. |
+| `error` | [`Error`](#error) | optional | In case of `"success": false`, this property holds information about the error that occurred. The singular form `error` should be used for a single error. |
```
Comment (ondrahermanek) [inline guidelines/responses.md:88]: @patrikmiksovsky95 single error in response is still supported? or is it rather deprecated? the doc examples (all 3 of them currently) show it as single item

image

Comment (patrikmiksovsky95) [reply guidelines/responses.md:88]: rather deprecated

Comment (MikeAdamsMews) [reply guidelines/responses.md:88]: Just to be clear, are we therefore saying that `error` as a single object is technically supported but **deprecated**, so API users should instead always be using the plural form `errors` and this should always be an array, even if there is a single error? So, for example, ```"error": [{"code":10, "message":"It ain't working"}]```?

Comment (ondrahermanek) [reply guidelines/responses.md:88]: @MikeAdamsMews yep

File: changelog/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 22nd April 2024
```
Comment (ondrahermanek) [inline changelog/README.md:3]: obsolete ...

File: guidelines/responses.md (hunk 1 above)
Comment (MikeAdamsMews) [reply guidelines/responses.md:88]: Thanks for the clarification @ondrahermanek, in that case I need to edit my changes to (i) mark 'error' (singular) as deprecated; (ii) remove 'error' (singular) from the examples and replace it with one for a singular error in 'errors' (plural).

**Review by ondrahermanek — APPROVED** [review]

**Review by ondrahermanek — APPROVED** [review]

**Pull request merged**